### DISCUSSION TITLE
zigbee_distinct_topics - SetOption89 as #define in user_config_override.h

### DISCUSSION THREAD

**Discussion by nagyrobi** [arendst/Tasmota#11195] [Ideas]
### PROBLEM DESCRIPTION
Have SetOption89 (Distinct MQTT topics per device for Zigbee) as configurable `#define` for `user_config_override.h`, in order to be able to compile firmware with that option set to `true` by default.
Currently this is not configurable through `user_config_override.h`.

### REQUESTED INFORMATION

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff ZbBridge Module
- [x] Tasmota binary firmware version number used: Tasmota 9.3.1.1
  - [ ] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: Tasmotizer / Web upgrade
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  not applicable


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  not applicable


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  not applicable


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  not applicable


```

### TO REPRODUCE
There's no `#define` for `zigbee_distinct_topics` in `my_user_config.h`.

### EXPECTED BEHAVIOUR
Have a `#define` for `zigbee_distinct_topics` in `my_user_config.h` set to `false`, which can be overridden in `user_config_override.h` with `true`.

### SCREENSHOTS
not applicable

### ADDITIONAL CONTEXT
None.

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by nagyrobi** [comment]
Also needed for SetOption73 and SetOption114: https://github.com/arendst/Tasmota/discussions/11194

**Comment by barbudor** [comment]
I'm on it
Any other one before I PR ?

**Reply by barbudor** [reply]
https://github.com/arendst/Tasmota/pull/11204
